packages/create-neutrino-project/commands/init/generator/writing.js

```diff
@@ -0,0 +1,158 @@
+const deepmerge = require('deepmerge');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const changeCase = require('change-case');
+const utils = require('./utils');
+
+const VERSIONS = utils.VERSIONS;
+const PROJECTS = utils.PROJECTS;
+const LINTERS = utils.LINTERS;
+
+function makeRcFile(data) {
+  let config = { use: [] };
+
+  if (data.projectType === PROJECTS.REACT) {
+    config.use.push('neutrino-preset-react');
+  } else {
+    config.use.push(`neutrino-preset-${data.projectType}`);
+  }
+
+  if (data.testRunner) {
+    config.use.push(`neutrino-preset-${data.testRunner}`);
+  }
+
+  switch (data.linter) {
+    case LINTERS.AIRBNB:
+      config.use.push(`neutrino-preset-${data.linter}${data.projectType === PROJECTS.WEB ? '-base' : ''}`);
+      break;
+    case LINTERS.STANDARDJS:
+      config.use.push(`neutrino-middleware-${LINTERS.STANDARDJS}`);
+      break;
+    default:
+      config.use.push(`neutrino-preset-${data.linter}`);
+  }
+
+  return `module.exports = ${JSON.stringify(config, null, 2)};\n`;
+}
+
+function requiredPackageJson(data) {
+  const devDependencies = {};
+  const dependencies = {};
+  const scripts = {};
+
+  switch (data.projectType) {
+    case PROJECTS.REACT:
+      dependencies[PROJECTS.REACT] = VERSIONS.REACT;
+      dependencies['react-dom'] = VERSIONS.REACT;
+      dependencies['react-hot-loader'] = VERSIONS.REACT_HOT_LOADER;
+      devDependencies['neutrino-preset-react'] = VERSIONS.NEUTRINO;
+      break;
+    case PROJECTS.REACT_COMPONENTS:
+      dependencies[PROJECTS.REACT] = VERSIONS.REACT;
+      dependencies['react-dom'] = VERSIONS.REACT;
+      dependencies['neutrino-preset-react-components'] = VERSIONS.REACT_COMPONENTS;
+      dependencies['react-addons-css-transition-group'] = VERSIONS.REACT_ADDONS_CSS_TRANSITION_GROUP;
+      break;
+    case PROJECTS.WEB:
+      devDependencies[`neutrino-preset-web`] = VERSIONS.NEUTRINO;
+      break;
+    case PROJECTS.NODE:
+      devDependencies[`neutrino-preset-node`] = VERSIONS.NEUTRINO;
+      break;
+    case PROJECTS.WEB_LIBRARY:
+      dependencies[`neutrino-preset-taskcluster-web-library`] = VERSIONS.WEB_LIBRARY;
+      break;
+  }
+
+  if (data.testRunner) {
+    devDependencies[`neutrino-preset-${data.testRunner}`] = VERSIONS.NEUTRINO;
+
+    if (data.testRunner === 'karma' || data.testRunner === 'mocha') {
+      devDependencies[`assert`] = VERSIONS.ASSERT;
+    }
+
+    scripts.test = 'neutrino test';
+  }
+
+  if (data.projectType === PROJECTS.REACT && data.router) {
+    dependencies['react-router-dom'] = VERSIONS.REACT_ROUTER_DOM;
+    dependencies['react-async-component'] = VERSIONS.REACT_ASYNC;
```

Comment:
I like react-loadable, and it works super-well with Webpack. Can we use that instead?